weaviate_cli/commands/revoke.py

```diff
@@ -0,0 +1,75 @@
+import click
```

Comment:
Aa a minor comment, I am not sure is needed a revoke command. I know there is a nuance with revoke and delete, but to me, it'd be more intuitive to have it under delete.

Just an option, no change needed if you don't agree :)

Reply:
This is something I was considering at the beginning, however I wanted to keep it consistent with the roles logic. A role can be assigned to a user, as a permission can be assigned to a role. The creation and deletion of the roles is handled by create and delete, but for the permissions there are no options to create them, they are pre-defined.
What I have done has been modifying the `add` and rename it by `assign`. After reading your comment I think it makes more sense.